so_pysm_models/extragalactic.py

```diff
@@ -0,0 +1,131 @@
+from . import InterpolatingComponent
+from . import utils
+import pysm
+import numpy as np
+import healpy as hp
+
+
+def y2uK_CMB(nu):
```

Comment:
@marcelo-alvarez can you add docstring with detail about this conversion?

Reply:
done.